src/app/code/community/Shopgate/Cloudapi/Model/Frontend/Observer/Layout.php

```diff
@@ -21,6 +21,16 @@
  */
 class Shopgate_Cloudapi_Model_Frontend_Observer_Layout
 {
+    /**
+     * Define custom handles
+     */
+    const CUSTOM_HANDLES = array(
+        array(
+            'path'   => '/customer/account/login',
+            'handle' => 'shopgate_cloudapi_customer_account_login'
```

Comment:
you also want to add the other handle ```shopgate_cloudapi_customer_customer_account_create``` ?

Reply:
'shopgate_cloudapi_customer_customer_account_create' is our own template / route. We only need the custom handle for default routes to inject JS or add / remove blocks. I'm sure that we need in the future other handles, but currently its only 'shopgate_cloudapi_customer_account_login' (customer_account_login)

Reply:
oh yeah, oops :)